src/tasks/installers/helm.ts

```diff
@@ -77,9 +77,13 @@ export class HelmTasks {
         task: async (ctx: any, task: any) => {
           const cheTlsSecret = await this.kubeHelper.getSecret(CHE_TLS_SECRET_NAME, flags.chenamespace)
 
+          const fixErrorMessage = 'Helm installer generates secrets automatically. To fix the problem delete existed secrets in dedicatedfor Eclispe Che namespace and rerun the connmand.'
```

Comment:
typo
`dedicatedfor`

Reply:
thanks

Reply:
`delete existed `che-tls` and `self-signed-certificate` secrets`